kubevela.yaml

```diff
@@ -47,19 +35,23 @@ pipeline:
       output: vela
       packages: ./references/cmd/cli/main.go
 
-  - uses: strip
```

Comment:
Should be ok to leave this step in - it is fairly conservative and will preserve symbol names in go binaries